pkg/operator/controllers/condition.go

```diff
@@ -37,30 +38,3 @@ func SetCondition(ctx context.Context, arocli aroclient.Interface, cond *status.
 		return err
 	})
 }
-
-func setStaticStatus(cluster *arov1alpha1.Cluster, role string) (changed bool) {
-	conditions := make(status.Conditions, 0, len(cluster.Status.Conditions))
-
-	// cleanup any old conditions
-	current := map[status.ConditionType]bool{}
-	for _, ct := range arov1alpha1.AllConditionTypes() {
-		current[ct] = true
-	}
```

Comment:
you will lose this functionality (above), I added this because at the time the conditions were "unstable". We were not sure if we would add/remove/rename any. So I added this so that there were not old conditions that were not getting updated. Maybe this is not needed anymore?

Reply:
So if we want to remove condition while removing some of the controllers we need to do this work. OR we need to replace this pattern where we clean conditions at some point... 
Sounds like this might be still needed IF we remove controllers.